probe-rs/examples/tcp_itm.rs

```diff
@@ -1,4 +1,4 @@
-use probe_rs::architecture::arm::swo::{Decoder, SwoPublisher, TracePacket, UpdaterChannel};
```

Comment:
maybe we should put those exports in a more accessible place I just regocnized ...